src/register-org/components/regulator-details/regulator-details.component.html

```diff
@@ -5,8 +5,8 @@
       <xuilib-hmcts-error-summary [errorMessages]="validationErrors" [header]="'There is a problem'"></xuilib-hmcts-error-summary>
       <span class="govuk-caption-xl">Register an organisation</span>
       <fieldset class="govuk-fieldset">
-        <legend class="govuk-fieldset__legend govuk-fieldset__legend--xl">
-          <h1 class="govuk-fieldset__heading">
+        <legend>
```

Comment:
@johnbenjamin-hmcts - may need reverting as per earlier comments

Reply:
@Daniel-Lam - I have updated the code now.